src/components/Card/IndicatorCard/IndicatorCard.js

```diff
@@ -38,6 +32,69 @@ const IndicatorCard = (props) => {
   const displayPrefix = `${indicator.cardPrefix || ''}${formatPrefix}`;
   const displaySuffix = `${formatSuffix}${indicator.cardSuffix || ''}`;
 
+  let numberType;
+  if (numberProps.showDualNumbers) {
+    if (numberProps.isSecondNumber) {
+      numberType = style.secondNumber;
+    } else {
+      numberType = style.firstNumber;
+    }
+  } else {
+    numberType = style.onlyNumber;
```

Comment:
Feels to me like this class, `onlyNumber`, is redundant.

Reply:
It technically is, but the CSS is so convoluted I felt it added a lot to the readability to set the font styles explicitly for 'only number' rather than set a general font size then override it for the small/large numbers.